nodewatcher/core/registry/fields.py

```diff
@@ -616,8 +616,6 @@ def get_queryset(self):
                     db = self._db or django_db.router.db_for_read(self.model, instance=self.instance)
                     empty_strings_as_null = django_db.connections[db].features.interprets_empty_strings_as_nulls
                     qs = super(RelatedManager, self).get_queryset()
-                    # Disable polymorphic queries as they can be very expensive in this case.
-                    qs = qs.non_polymorphic()
```

Comment:
This needs to be removed, otherwise you will only get top-level items. I am not sure whether we can make it perform any better as the previous comment indicated that this is a major performance hit.